src/app/components/nav/simple-header/simple-header.component.ts

```diff
@@ -0,0 +1,150 @@
+import {ChangeDetectorRef, Component, ElementRef, HostListener, Inject, OnDestroy, OnInit, ViewChild} from '@angular/core';
+import {GlobalService} from '../../../services/global.service';
+import {ActivatedRoute, Router} from '@angular/router';
+import {AuthService} from '../../../services/auth.service';
+import {ApiService} from '../../../services/api.service';
+import {DOCUMENT} from '@angular/common';
+
+@Component({
+  selector: 'app-simple-header',
+  templateUrl: './simple-header.component.html',
+  styleUrls: ['./simple-header.component.scss']
+})
+export class SimpleHeaderComponent implements OnInit, OnDestroy {
+
+  user = {username: ''};
+
+  /**
+   * Header white flag
+   */
+  headerWhite = false;
+
+  /**
+   * Header is transparent on these URLs
+   */
+  transparentHeaderUrls = ['', '/'];
+
+  /**
+   * Is router at '/'
+   */
+  atHome = true;
+
+  /**
+   * Scroll position
+   */
+  scrolledState = false;
+
+  /**
+   * Is header menu expanded
+   */
+  isMenuExpanded = true;
+
+  /**
+   * Global Service subscription
+   */
+  globalServiceSubscription: any;
+
+  /**
+   * Authentication Service subscription
+   */
+  authServiceSubscription: any;
+
+  /**
+   * Current Authentication state
+   */
+  authState: any;
+
+  /**
+   * Inner width
+   */
+  public innerWidth: any;
```

Comment:
(**minor**) This could be typed as a number, right? (Similar type changes could be done at some other places.)